scripts/ci/automation_full_test.py

```diff
@@ -36,6 +36,7 @@
 pull_request_number = os.environ.get('PULL_REQUEST_NUMBER', None)
 enable_pipeline_result = bool(job_name and python_version)
 unique_job_name = ' '.join([job_name, python_version, profile, str(instance_idx)]) if enable_pipeline_result else None
+enable_traceback = True if os.environ.get('ENABLE_TRACEBACK').lower() == 'true' else False
```

Comment:
Traceback is not enabled by default?

Reply:
Good question, this default value can be configured directly in the pipeline.
Whether it's enable or disable by default, we can override it when running manually.
![image](https://github.com/Azure/azure-cli/assets/18628534/176e3a70-87c5-45ca-8758-40001040e2a5)